src/README.rst

```diff
@@ -16,11 +16,14 @@ To get started, after installation run the following:
 Change Log
 ==========
 
+3.2.0
+
 3.1.0
 -----
 
 - Update to official 6.0.2 Service Fabric SDK
 - Property command group added
+- ChaosContext (context) and ChaosTargetFilter (chaos-target-filter) arguments are added to Chaos start command (#62)
```

Comment:
Forgot to ask last time, are the chaos changes breaking changes? If so, we should make this version 4.0.0. If not, then this is good. Thanks!

Reply:
This is not breaking change.